### ISSUE TITLE
[Feature] Explain Problem popup

### ISSUE THREAD

**Issue by erikdemarco** [continuedev/continue#2309]
### Validations

- [X] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

In codeium there is a very helpful feature called "explain problem popup". When there is an error it can show popup when hovering to the problem, and when clicked it automatically ask an AI to explain the problem. In "Continue" currently its impossible to do it, even when you manually ask the AI to explain the problem by include current file context and the line number, it always wrongly pinpoint the exact line. Maybe the Continue plugin doing some trimming so the exact line always is incorrect.

### Solution

Please add the feature, or at least fix the bug which always causes exact line to pinpoint incorrectly

**Comment by Patrick-Erichsen** [comment]
Hi @erikdemarco , do you have a docs link for the Codeium feature you're describing? I would be interested in taking a closer look at it.

I believe you can accomplish what you're describing in Continue:
![Screenshot 2024-09-19 at 11 54 05 AM](https://github.com/user-attachments/assets/8ffb20d3-1ca7-43a9-a991-2b5518ca62fa)
![Screenshot 2024-09-19 at 11 54 11 AM](https://github.com/user-attachments/assets/74abec9f-29c4-4a27-a47c-9ee6010b5e9d)

**Comment by erikdemarco** [comment]
@Patrick-Erichsen yes this is what im looking for. thank you very much. But is it possible to auto include "currently active document" context instead of just the snippet of the problem? Or is there any setting to auto include current active document for current project? in codeium current document is auto included in any chat. and its easy to autoinclude codebase in current project.

**Comment by Patrick-Erichsen** [comment]
Good question, unfortunately there isn't at the moment. The closest you could get would be to cancel the request, and then resubmit the prompt using `option + enter` to use the current file.

Could you share more around how you were using current active file with Codeium? Did you have some configuration that included the active file for all new chats, including chat started from tools like the "Quick Fix" element here?
